nflows/transforms/standard.py

```diff
@@ -30,15 +30,15 @@ def __init__(
         super().__init__()
         shift, scale = map(torch.as_tensor, (shift, scale))
 
-        if not (scale > 0.0).all():
-            raise ValueError("Scale must be strictly positive.")
+        if (scale == 0.0).any():
+            raise ValueError("Scale must be non-zero.")
 
         self.register_buffer("_shift", shift)
         self.register_buffer("_scale", scale)
 
     @property
     def _log_scale(self) -> Tensor:
```

Comment:
Can we rename this to `_log_abs_scale` to avoid confusion?

Reply:
Done!